src/getHumanAge.test.js

```diff
@@ -7,4 +7,39 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return 0 for cat and dog if input values equals 0', () => {
```

Comment:
```suggestion
  test('should return [0, 0] if input values are (0, 0)', () => {
```
because now it seems like the function returns a number and accepts one number. fix everywhere